data/ecosystems/s/stacks.toml

```diff
@@ -4487,3 +4487,148 @@ url = "https://gitlab.com/steve-dejo/animo"
 
 [[repo]]
 url = "https://gitlab.com/vaultilo/vaultilo-app"
+
+
+[[repo]]
+url = "https://github.com/Satoshiaaw2/HiroTask1"
+
+[[repo]]
+url = "https://github.com/Stacker2023/FriendTech"
+
+[[repo]]
+url = "https://github.com/nescampos/friendtech_clone_stacks"
+
+[[repo]]
+url = "https://github.com/The-Six/Hiro-Friend.tech2.git"
+
+[[repo]]
+url = "https://github.com/nochiel/hiro-hacks-friendtech"
+
+[[repo]]
+url = "https://github.com/Oyeins-GUI/friend.tech"
+
+[[repo]]
+url = "https://github.com/Aman-zishan/friend.tech"
+
+[[repo]]
+url = "https://github.com/davek-btc/Friend.tech/blob/main/contracts/keys.clar"
+
+[[repo]]
+url = "https://github.com/frodoman/hiro-hack-one"
+
+[[repo]]
+url = "https://github.com/vicnicius/friend-tech-stx"
+
+[[repo]]
+url = "https://github.com/natebuch/hirohacksubmissionwk1"
+
+[[repo]]
+url = "https://github.com/RedVelvetZip/hiro-hacks"
+
+[[repo]]
+url = "https://github.com/tippenein/friend-tech"
+
+[[repo]]
+url = "https://github.com/codeonedotzero/hiro-hack-1"
+
+[[repo]]
+url = "https://github.com/Satoshiaaw2/HiroTask2"
+
+[[repo]]
+url = "https://github.com/nescampos/dgrant_stacks_clone"
+
+[[repo]]
+url = "https://github.com/The-Six/Hiro-GrantsProgram.git"
+
+[[repo]]
+url = "https://github.com/Stacker2023/DecentralizedGrants"
+
+[[repo]]
+url = "https://github.com/alexngys/Clarify"
+
+[[repo]]
+url = "https://github.com/livetreetech/Stacks"
+
+[[repo]]
+url = "https://github.com/Podima2/Mycelium"
+
+[[repo]]
+url = "https://github.com/djleo80/Valeur---An-Assset-Management-System"
+
+[[repo]]
+url = "https://github.com/play3rLeaders"
+
+[[repo]]
+url = "https://github.com/edsml-cc2823/bitroll"
+
+[[repo]]
+url = "https://github.com/jjjutla/Auto-BTC"
+
+[[repo]]
+url = "https://github.com/sgno28/liquid-stacking-sbtc"
+
+[[repo]]
+url = "https://github.com/indisofyar/Seed-Garden.git"
+
+[[repo]]
+url = "https://github.com/SamyHub/clarify-ai"
+
+[[repo]]
+url = "https://github.com/bilal-mustafa10/stack-backend"
```

Comment:
not sure this is part of Stacks?

Reply:
This is an AI tool for generating Clarity code. The stack-backend functionality for generating Clarity code is in the main.py file.